README.md

```diff
@@ -74,6 +74,7 @@ services:
     container_name: flame
     volumes:
       - <host_dir>:/app/data
+      - /var/run/docker.sock:/var/sock/docker.sock # Docker socket
```

Comment:
No, is only required for getting the docker containers.
And I just saw that I have to change the code to prevent the api calling without having that file.

Reply:
Ok, so maybe add some comment saying this mount is optional or only required for this feature.